src/wepy/resampling/novelty/novelty.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+class Novelty():
+
+    cycle_counter = 0
+    clone_merge_iter = 0
+    sample_counter = 0
+
+    def __init__(self):
+        pass
+
+    def calculate_novelty(self, image = None, amp = None, weight = None):
```

Comment:
docstring. Fill in the arguments docs

Reply:
Added doc strings to the calculate_novelty method.